dispatcher/dispatcher.go

```diff
@@ -1245,3 +1248,66 @@ func (d *Dispatcher) setNodeRole(id NodeId, role OtDeviceRole) {
 	node.Role = role
 	d.vis.SetNodeRole(id, role)
 }
+
+func (d *Dispatcher) handleCoapEvent(node *Node, argsStr string) {
+	var err error
+
+	if d.coaps == nil {
+		// Coaps not enabled
+		return
+	}
```

Comment:
Show the function be renamed to `handleCoapsEvent`?
```suggestion
func (d *Dispatcher) handleCoapsEvent(node *Node, argsStr string) {
	var err error

	if d.coaps == nil {
		// Coaps not enabled
		return
	}
```

Reply:
The event is `coap` event. 
The handler collects multiple `coap` events, thus it's called `coaps`